### DISCUSSION TITLE
Crypto to alternative currency (csv upload)

### DISCUSSION THREAD

**Discussion by bjk201** [ghostfolio/ghostfolio#4098] [Q&A]
Hi everyone, I recently started using Ghostfolio and am currently importing my transactions. My main currency is EUR. I have taken my data from two exchanges and copied it into the CSV template.
Since my export contains the EURO amounts, I have also entered EUR in the upload. When importing, I now get the message that my currency created in Ghostfolio (e.g. Bitcoin) cannot be imported as EUR. Unfortunately, it is also not possible to create BTCEUR that GF loads the appropriate euro history data.
Is there a possibility to add another “data provider”? 
Thank you

Translated with DeepL.com (free version)

**Comment by dtslvr** [comment]
Hi @bjk201

Thanks for starting this discussion! It seems related to https://github.com/ghostfolio/ghostfolio/issues/3320, so I will close this discussion to keep everything in one place.
